android/app/src/main/kotlin/com/jabook/app/jabook/audio/MediaSourceValidator.kt

```
@@ -0,0 +1,192 @@
+            header[1] == 'L'.code.toByte() &&
+            header[2] == 'a'.code.toByte() &&
+            header[3] == 'C'.code.toByte()
+        ) {
+            return true
+        }
+
+        // WAV: "RIFF"
+        if (header[0] == 'R'.code.toByte() &&
+            header[1] == 'I'.code.toByte() &&
+            header[2] == 'F'.code.toByte() &&
+            header[3] == 'F'.code.toByte()
```

detekt (1.23.8)

[warning] This condition is too complex (4). Defined complexity threshold for conditions is set to '4'

(detekt.complexity.ComplexCondition)